terraform/scripts/lint.sh

```diff
@@ -16,7 +16,7 @@ chmod +x "${WORKING_DIR}"/terraform-provider-pingdom
 
 cd "${WORKING_DIR}"
 
-for dir in ${PAAS_CF_DIR}/terraform/*/ ; do
+for dir in "${PAAS_CF_DIR}"/terraform/*/ ; do
```

Comment:
Same as above - I'd prefer the whole thing was in quotes.

Reply:
As above; the shell interpolation gets in the way if you put the whole line in quotes.